### DISCUSSION TITLE
[Feature Request] A command to toggle pvp for yourself or others if you're op.

### DISCUSSION THREAD

**Discussion by AZFire1** [Zrips/CMI#4258] [Feature Requests]
**Description of feature/improvement request:**
The basic description of it (if this doesn't already exist, as I might've overlooked it) is that you can toggle pvp for yourself. If you're op you could toggle pvp for others, this could be very useful if it's not already a thing.
---
### Detailed explanation of what it should do, when and in what situation: ###
The explanation of this feature is bassically to be able to toggle pvp.
### Basic explanation as to why this feature is needed and/or useful to have: ###
This feature could definitely be useful for survival servers that aren't really pvp based so players that want to turn this off and enjoy survival can have it off, and the players that are more into pvp can have it on.

**Comment by ChiboYen** [comment]
Should be a permissions based command, but otherwise this is a nice addition to CMI

**Comment by CloudeLecaw** [comment]
Duplicate request with simliar vantage from here

#4040 , #2249 , #2360
